csrc/sparse/cutlass/sparse_compressor_c3x.cu

```diff
@@ -2,6 +2,7 @@
 // clang-format off
 #include <cudaTypedefs.h>
 
+#if defined CUDA_VERSION && CUDA_VERSION >= 12020
```

Comment:
Dumb q: 12020 is right for 12.2? I guess if 0 is a period

Reply:
yeah, you can see where we check for 12.4 here:

https://github.com/vllm-project/vllm/blob/fdea8ec16775e1645620b5ff46b799d60df4624c/csrc/quantization/cutlass_w8a8/scaled_mm_entry.cu#L73-L79